<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_Data">
     <property name="title">
      <string>Load Data</string>
     </property>
     <addaction name="actionLoad_Video"/>
     <addaction name="actionLoad_Images"/>
     <addaction name="actionLoad_Intervals"/>
     <addaction name="actionLoad_Events"/>
     <addaction name="actionLoad_Points"/>
     <addaction name="actionLoad_Lines"/>
     <addaction name="actionLoad_Masks"/>
     <addaction name="actionLoad_Tensor"/>
     <addaction name="actionLoad_JSON_Config"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_Video"/>
     <addaction name="actionExport_Spreadsheet"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuLoad_Data"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionTake_Snapshot"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Modules</string>
    </property>
     <addaction name="actionTable_Designer"/>
    <addaction name="actionData_Manager"/>
    <addaction name="actionGroup_Management"/>
    <addaction name="actionBatch_Processing"/>
    <addaction name="actionWhisker_Tracking"/>
    <addaction name="actionTongue_Tracking"/>
    <addaction name="actionMachine_Learning"/>
    <addaction name="actionData_Viewer"/>
    <addaction name="actionNew_Media_Widget"/>
    <addaction name="actionData_Transforms"/>
    <addaction name="actionTerminal_Output"/>
    <addaction name="actionAnalysis_Dashboard"/>
    <addaction name="actionTest_Widget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuModules"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTake_Snapshot">
   <property name="text">
    <string>Take Snapshot</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionWhisker_Tracking">
   <property name="text">
    <string>Whisker Tracking</string>
   </property>
  </action>
  <action name="actionLabel_Maker">
   <property name="text">
    <string>Label Maker</string>
   </property>
  </action>
  <action name="actionLoad_Images">
   <property name="text">
    <string>Load Images</string>
   </property>
  </action>
  <action name="actionAnalog_Viewer">
   <property name="text">
    <string>Analog Viewer</string>
   </property>
  </action>
  <action name="actionImage_Processing">
   <property name="text">
    <string>Image Processing</string>
   </property>
  </action>
  <action name="actionTongue_Tracking">
   <property name="text">
    <string>Tongue Tracking</string>
   </property>
  </action>
  <action name="actionLoad_Analog_Time_Series_CSV">
   <property name="text">
    <string>Load Analog Time Series (CSV)</string>
   </property>
  </action>
  <action name="actionLoad_Digital_Time_Series_CSV">
   <property name="text">
    <string>Load Digital Time Series (CSV)</string>
   </property>
  </action>
  <action name="actionLoad_Video_mat_hdf5">
   <property name="text">
    <string>Load Video (.mat, .hdf5)</string>
   </property>
  </action>
  <action name="actionLoad_Video">
   <property name="text">
    <string>Load Video</string>
   </property>
  </action>
  <action name="actionTracking_Widget">
   <property name="text">
    <string>Tracking Widget</string>
   </property>
  </action>
  <action name="actionMachine_Learning">
   <property name="text">
    <string>Machine Learning</string>
   </property>
  </action>
  <action name="actionData_Viewer">
   <property name="text">
    <string>Data Viewer</string>
   </property>
  </action>
  <action name="actionNew_Media_Widget">
   <property name="text">
    <string>New Media Widget</string>
   </property>
  </action>
  <action name="actionBatch_Processing">
   <property name="text">
    <string>Batch Processing</string>
   </property>
  </action>
  <action name="actionLoad_JSON_Config">
   <property name="text">
    <string>Load JSON Config</string>
   </property>
  </action>
  <action name="actionLoad_Points">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionLoad_Masks">
   <property name="text">
    <string>Load Masks</string>
   </property>
  </action>
  <action name="actionLoad_Lines">
   <property name="text">
    <string>Load Lines</string>
   </property>
  </action>
  <action name="actionLoad_Intervals">
   <property name="text">
    <string>Load Intervals</string>
   </property>
  </action>
  <action name="actionData_Manager">
   <property name="text">
    <string>Data Manager</string>
   </property>
  </action>
  <action name="actionGroup_Management">
   <property name="text">
    <string>Group Management</string>
   </property>
  </action>
  <action name="actionLoad_Tensor">
   <property name="text">
    <string>Load Tensor</string>
   </property>
  </action>
  <action name="actionExport_Video">
   <property name="text">
    <string>Export Video</string>
   </property>
  </action>
  <action name="actionData_Transforms">
   <property name="text">
    <string>Data Transforms</string>
   </property>
  </action>
  <action name="actionLoad_Events">
   <property name="text">
    <string>Load Events</string>
   </property>
  </action>
  <action name="actionExport_Spreadsheet">
   <property name="text">
    <string>Export Spreadsheet</string>
   </property>
  </action>
  <action name="actionTerminal_Output">
   <property name="text">
    <string>Terminal Output</string>
   </property>
  </action>
  <action name="actionAnalysis_Dashboard">
   <property name="text">
    <string>Analysis Dashboard</string>
   </property>
  </action>
  <action name="actionTable_Designer">
   <property name="text">
    <string>Table Designer</string>
   </property>
  </action>
  <action name="actionTest_Widget">
   <property name="text">
    <string>Test Widget</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
